script/Ciel/Day 14/14_01B_02_CIEL14B_3B.txt

```diff
@@ -387,12 +391,17 @@ That thing doesn't feed on protein,
 [sha:dc4be5129368a9a8682f2c09e8fe087037c9afac]{
 -- Page 153, Offset 39853.
 -- 「“もう色々びしょびしょね。
+// What did she mean by this
+%{i}
 "Looks like you're already wrung dry, aren't you?
+%{/i}
```

Comment:
As I said, let's just keep it fairly literal.
I haven't really found anyone else who can divine the purpose here.

Reply:
> Just look at what a mess you already are.

?

Reply:
Sure, I guess